etc/DFLMessages_en_GB.dic

```diff
@@ -37,16 +37,25 @@ RefUnsupportedTag             =     Settings ref %1% uses a unrecognized tag %2%
 
 //------------------ Main ---------------------------
 NetworkSlackNodeNotFound      =     Network slack node requested but not found in network input file %1%
-InputsInfo                    =     Processing network input file %1% and config file %2%
+InputsSteadyStateInfo         =     Processing network input file %1% and config file %2% for steady-state calculation
```

Comment:
Rename these keys: starting with "Inputs" is confusing since it is associated with an input log

Reply:
Done. Removed the "Inputs" prefix.